src/Category/Construct/Agda.agda

```diff
@@ -0,0 +1,67 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Agda as a category
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Category.Construct.Agda where
+
+open import Level
+open import Category.Category
+open import Function
+open import Relation.Binary.PropositionalEquality
+
+agdaCat : ∀ ℓ → Category ℓ
+agdaCat ℓ = record
+  { Obj = Set ℓ
+  ; _⇒_ = λ A B → A → B
+  ; _≈_ = _≗_
+  ; id = Function.id
+  ; _∘_ = Function._∘′_
+  }
+
+-- these construction recover the old definitions of RawFunctor and Morphism
+-- and provide a way to upgrade them to the generalized version
+module _ {ℓ : Level} where
+  open import Category.Functor (agdaCat ℓ) (agdaCat ℓ)
```

Comment:
These imports should go at the top of the file, and be given aliases that can then be opened here.